.github/workflows/jlpkgbutler-ci-pr-workflow.yml

```diff
@@ -9,7 +9,9 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        julia-version: ['1.0', '1.6', '1.9', 'nightly']
+        # FIXME: temporarily removing nightly due to failing build.
```

Comment:
Should we revert this before merging, now that the tests have passed?

Reply:
Yeah, let's.

Reply:
Done.